<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAgregarCliente</class>
 <widget class="QDialog" name="DialogAgregarCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Agregar Cliente</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>31</y>
        <width>44</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nombre">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>31</y>
        <width>108</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>61</y>
        <width>44</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Apellido</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>91</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tel</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>181</y>
        <width>166</width>
        <height>24</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Apellido">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>61</y>
        <width>108</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Telefono">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>91</y>
        <width>108</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>121</y>
        <width>34</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>E-Mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Email">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>121</y>
        <width>108</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Direccion">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>151</y>
        <width>108</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>151</y>
        <width>50</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Direccion</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Nombre</tabstop>
  <tabstop>lineEdit_Apellido</tabstop>
  <tabstop>lineEdit_Telefono</tabstop>
  <tabstop>lineEdit_Email</tabstop>
  <tabstop>lineEdit_Direccion</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
